### PR TITLE
:link: Create parsing for the start of the server

### PR DESCRIPTION
**Pull request by Peralban**
I invite you to see this issues for more informations:
- #6 

ATM the structure of the server is that
image

### CODE REVIEW

**Review by Peralban — COMMENTED** [review]
Normally it's good

File: Network/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.10)
```
Comment (Antoinegtir) [inline Network/CMakeLists.txt:1]: Don't understand the purpose of the cmake

File: Network/Makefile_Network.mk [hunk 2]
```diff
@@ -27,7 +32,7 @@ INCLUDE		=	-I./include
 
 VALGRIND	= -g3
 
-CFLAGS	=	$(INCLUDE) $(WARNINGS) #$(VALGRIND)
+CFLAGS	=	$(INCLUDE) $(WARNINGS) $(VALGRIND)
```
Comment (Antoinegtir) [inline Network/Makefile_Network.mk:35]: comment `VALGRIND` flag when push in the main branch

File: Network/include/lib/my.h [hunk 3]
```diff
@@ -0,0 +1,71 @@
+/*
+** EPITECH PROJECT, 2022
+** lib
+** File description:
+** my.h
+*/
+
+#pragma once
+
+#define ARRAY 0
```
Comment (Antoinegtir) [inline Network/include/lib/my.h:10]: Macro `ARRAY` use nowhere

**Review by Antoinegtir — CHANGES_REQUESTED** [review]
great job, there is small issues to solve

File: Network/CMakeLists.txt (hunk 1 above)
Comment (Peralban) [reply Network/CMakeLists.txt:1]: It's for people who use Clion

**Pull request merged**